addons/calendar/models/calendar_event.py

```diff
@@ -629,19 +629,15 @@ def write(self, values):
 
         current_attendees = self.filtered('active').attendee_ids
         if 'partner_ids' in values:
-            # we send to all partners and not only the new ones
-            (current_attendees - previous_attendees)._send_mail_to_attendees(
-                self.env.ref('calendar.calendar_template_meeting_invitation', raise_if_not_found=False)
-            )
+            self._send_template_to_attendees('calendar.calendar_template_meeting_invitation', attendees=current_attendees - previous_attendees)
         if not self.env.context.get('is_calendar_event_new') and 'start' in values:
             start_date = fields.Datetime.to_datetime(values.get('start'))
             # Only notify on future events
             if start_date and start_date >= fields.Datetime.now():
-                (current_attendees & previous_attendees).with_context(
+                recipient_attendees = current_attendees & previous_attendees
+                self.with_context(
                     calendar_template_ignore_recurrence=not update_recurrence
-                )._send_mail_to_attendees(
-                    self.env.ref('calendar.calendar_template_meeting_changedate', raise_if_not_found=False)
-                )
+                )._send_template_to_attendees('calendar.calendar_template_meeting_changedate', recipient_attendees)
```

Comment:
`attendees=recipient_attendees` :socks: 
EDIT: or perhaps not, see last comment ^^